src/count/README.md

```diff
@@ -0,0 +1,172 @@
+# o-comment-count [![Build Status](https://travis-ci.org/Financial-Times/o-comment-count.svg?branch=master)](https://travis-ci.org/Financial-Times/o-comment-count) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
+
+A small UI component for rendering the comment count of a given article.
+
+## Contents
+
+ * <a href="#product">Adding comment count to your product</a>
+     * <a href="#decl">Declaratively</a>
+     * <a href="#imper">Imperatively</a>
+ * <a href="#configuration">Global configuration</a>
+     * <a href="#confdecl">Declaratively</a>
+     * <a href="#confimper">Imperatively</a>
+ * <a href="#jsapi">JavaScript API</a>
+     * <a href="#methods">Methods</a>
+     * <a href="#logging">Logging</a>
+ * <a href="#browser">Browser support</a>
+ * <a href="#core">Core/enhanced experience</a>
+
+## <div id="product"></div> Adding comment count to your product
+### <div id="decl"></div> Declaratively
+Use the following markup to enable comments:
+
+```html
+<div data-o-component="o-comment-count"
```

Comment:
Should we reflect the namespace changes in the readme as well?

Reply:
Yes, thanks for catching that!